### DISCUSSION TITLE
Change password?

### DISCUSSION THREAD

**Discussion by daninfpj** [codemagic-ci-cd/codemagic-docs#2689] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

Where can I update my account password? Email? Set up 2FA?

I go to “Your Profile”… nothing there.
I go to “Billing Settings” and from there I can access “User Settings”!… but still nothing.

### Expected results

An account settings section with security settings. Basic stuff.

### Actual results

There is nothing like that.

Also the navigation seems very unintuitive: User preferences > Personal account billing details > Personal account settings.

**Comment by dtrdic** [comment, marked as answer]
Hello @daninfpj, unfortunately the login email cannot be changed. But, you can transfer the app to a new email and change the app ownership if you need.

Do note that you will need to recreate the environment variables in that case.
Anyway, if you're comfortable with this solution, these are the steps to follow:

```
1. Create a new account using a new email
2. Create a team or add all the necessary apps to your existing team
3. Invite the new account/person to the team
4. Promote them to the owner status
5. Update all necessary credentials if required
6. Remove the old user from the team
```

Regarding the password change.. Each time you log in to your Codemagic account via email, a new `One Time Password` is sent to your email address, so there’s no permanent password associated with your account.
It’s also possible to enable multi-factor authentication (MFA) on the OAuth provider side. For example, you can enable MFA on GitHub and sign up via Github on Codemagic.

**Comment by daninfpj** [comment]
Got it, thanks!
